src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+  const LONG_TERM = 7;
+  const SHIRT_TERM = 3;
+  const LONG_TERM_DISKONT = 50;
+  const SHIRT_TERM_DISKONT = 20;
+  const PRICE_RENT = 40;
+
+  if (days < SHIRT_TERM) {
+    return (total += days * PRICE_RENT);
```

Comment:
There's no need to use the assignment operator here. You can return the result of the calculation directly.